### PR TITLE
Port Gstreamer related functionality

### PR DESCRIPTION
**Pull request by anosach-luxoft**

### CODE REVIEW

**Comment by anosach-luxoft** [comment]
@malirod, @pestOO, @nk0leg please review

File: src/components/media_manager/src/audio/from_mic_to_file_recorder_thread.cc [hunk 1]
```diff
@@ -31,10 +31,26 @@
...
+
 #include <sstream>
 #include "utils/logger.h"
 
+namespace {
+  gboolean ReceiveData(GstBus* bus, GstMessage* message, gpointer data) {
+#if defined(OS_POSIX)
+    return recvmsg(bus, message, data);
+#elif defined(OS_WINDOWS)
+    return true;
```
Comment (pestOO) [inline src/components/media_manager/src/audio/from_mic_to_file_recorder_thread.cc:72]: Is it workaround?

Comment (anosach-luxoft) [inline src/components/media_manager/src/audio/from_mic_to_file_recorder_thread.cc:72]: Yeah :smile:

Comment (pestOO) [inline src/components/media_manager/src/audio/from_mic_to_file_recorder_thread.cc:72]: Please, add comment in future.

**Comment by pestOO** [comment]
Reviewed

File: src/components/media_manager/src/audio/from_mic_to_file_recorder_thread.cc (hunk 1 above)
Comment (anosach-luxoft) [inline src/components/media_manager/src/audio/from_mic_to_file_recorder_thread.cc:72]: @pestOO see b23f3dc

**Comment by anosach-luxoft** [comment]
@malirod please review

**Comment by anosach-luxoft** [comment]
@malirod AFAIK in license header should be enumerated all years when file was updated. So my way is correct. If you don't agree please give me a proof and I'll change it.

**Comment by anosach-luxoft** [comment]
@malirod see 153d3cf

**Comment by anosach-luxoft** [comment]
Find modules should work according to https://cmake.org/Wiki/CMake:How_To_Find_Libraries and should not be different for all platforms. These issues will be fixed within #60.

**Pull request merged**